src/main/java/org/folio/circulation/services/ItemsInTransitReportService.java

```diff
@@ -115,9 +128,17 @@
   }
 
   private CompletableFuture<Result<ItemsInTransitReportContext>> fetchUsers(
-    Result<ItemsInTransitReportContext> context) {
+    ItemsInTransitReportContext context) {
 
-    return completedFuture(context);
+    final Result<CqlQuery> cqlQuery = exactMatchAny("status", openStates());
+    FindWithMultipleCqlIndexValues<Request> fetcher
+      = findWithMultipleCqlIndexValues(requestsStorageClient, "requests",
+      Request::from);
+
+    return fetcher.findByIdIndexAndQuery(context.getItems().keySet(), "itemId", cqlQuery)
```

Comment:
Why are we fetching requests again? They should be already fetched and put in the context by this point (fetchRequests(Result<ItemsInTransitReportContext> context))

Reply:
yeah, it was redundant, fixed